.buildkite/pipeline.yml

```
@@ -70,6 +70,30 @@ steps:
                     - BUILDKITE_PULL_REQUEST=${BUILDKITE_PULL_REQUEST}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
 
+  - label: "Trigger Metricbeat"
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only ${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          watch:
+            - path:
+                - metricbeat/
+                - .buildkite/metricbeat/
+                - .buildkite/scripts
+                #OSS
+                - go.mod
+                - pytest.ini
+                - dev-tools/**
```

I see in other PRs that this was adjusted to 

```suggestion
                - dev-tools/
```

@pazone can give some insight into why; better to keep it consistent.